cpp/log/sqlite_db.h

```diff
@@ -2,6 +2,7 @@
 #ifndef SQLITE_DB_H
 #define SQLITE_DB_H
 
+#include <memory>
```

Comment:
I was previously using a unique_ptr for something, but then removed it - dinged.